doc/lsp/zed.rst

```diff
@@ -0,0 +1,6 @@
+.. _lsp_client_zed:
+
+Zed
+================
+
+phpactor is default LSP for zed. Install zed's `Php extension <https://github.com/zed-industries/zed/tree/main/extensions/php/>`_.
```

Comment:
`Phpactor is the default LSP for Zed. Install Zed's` then PHP should be all caps: `PHP extension`